cluster/src/test/java/com/hartwig/pipeline/calling/structural/gridss/stage/GridssSubStageTest.java

```diff
@@ -0,0 +1,29 @@
+package com.hartwig.pipeline.calling.structural.gridss.stage;
+
+import static org.junit.Assert.fail;
+
+import com.hartwig.pipeline.calling.SubStageTest;
+import com.hartwig.pipeline.calling.structural.gridss.CommonEntities;
+import com.hartwig.pipeline.execution.vm.BashCommand;
+
+import org.mockito.ArgumentCaptor;
+
+public abstract class GridssSubStageTest extends SubStageTest implements CommonEntities {
+    ArgumentCaptor<BashCommand> captor;
+
+    void assertBashContains(final BashCommand model) {
```

Comment:
prefer not to use a class hierarchy to share code, maybe extract into BashAssertions class? Might be handy elsewhere.